tests/fixtures/use-cache/app/next-cache.ts

```diff
@@ -0,0 +1,20 @@
+import * as NextCacheTyped from 'next/cache'
+
+const NextCache = NextCacheTyped as any
+
+export const cacheLife: any =
+  'cacheLife' in NextCache
+    ? NextCache.cacheLife
+    : 'unstable_cacheLife' in NextCache
+      ? NextCache.unstable_cacheLife
+      : () => {
+          throw new Error('both unstable_cacheLife and cacheLife are missing from next/cache')
+        }
```

Comment:
This was changed in https://github.com/vercel/next.js/pull/84877

The `cacheTag` handling is just future proofing (it's still `unstable_cacheTag` as of now)